### PR TITLE
Project refactoring

### PR DESCRIPTION
**Pull request by st61014**

### CODE REVIEW

File: NNPTPZ1/ComplexNumber.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        {
+            if (obj is ComplexNumber)
+            {
+                ComplexNumber numberForComparision = obj as ComplexNumber;
+                return numberForComparision.RealPart == RealPart && numberForComparision.ImaginaryPart == ImaginaryPart;
+            }
+            return false;
+        }
+
+        public readonly static ComplexNumber Zero = new ComplexNumber()
```
Comment (rodi0878) [inline NNPTPZ1/ComplexNumber.cs]: Properties should be at the top of class

File: NNPTPZ1/ComplexNumber.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        public ComplexNumber Multiply(ComplexNumber argumentComplexNumber)
+        {
+            ComplexNumber baseComplexNumber = this;
+            return new ComplexNumber()
+            {
+                RealPart = baseComplexNumber.RealPart * argumentComplexNumber.RealPart - baseComplexNumber.ImaginaryPart * argumentComplexNumber.ImaginaryPart,
+                ImaginaryPart = baseComplexNumber.RealPart * argumentComplexNumber.ImaginaryPart + baseComplexNumber.ImaginaryPart * argumentComplexNumber.RealPart
+            };
+        }
+        public double GetAbS()
```
Comment (rodi0878) [inline NNPTPZ1/ComplexNumber.cs]: Ab"S"?

File: NNPTPZ1/ComplexNumber.cs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        public ComplexNumber Add(ComplexNumber argumentComplexNumber)
+        {
+            ComplexNumber baseComplexNumber = this;
+            return new ComplexNumber()
+            {
+                RealPart = baseComplexNumber.RealPart + argumentComplexNumber.RealPart,
+                ImaginaryPart = baseComplexNumber.ImaginaryPart + argumentComplexNumber.ImaginaryPart
+            };
+        }
+        public double GetAngleInDegrees()
```
Comment (rodi0878) [inline NNPTPZ1/ComplexNumber.cs]: Really degrees?

**Review by rodi0878 — CHANGES_REQUESTED** [review]
Please look at comments and provide update

**Comment by st61014** [comment]
requested changes resolved

**Comment by rodi0878** [comment]
Accepted

**Pull request closed without merging**